pybtp/btp.py

```diff
@@ -2031,6 +2091,40 @@ def gattc_disc_prim_uuid_find_attrs_rsp(exp_svcs, store_attrs=False):
                 GATT_SVCS.append(svc)
 
 
+def gattc_disc_all_prim_rsp(store_rsp=False):
+    iutctl = get_iut()
+
+    tuple_hdr, tuple_data = iutctl.btp_socket.read()
+    logging.debug("%s received %r %r", gattc_disc_all_prim_rsp.__name__,
+                  tuple_hdr, tuple_data)
+
+    btp_hdr_check(tuple_hdr, defs.BTP_SERVICE_ID_GATT,
+                  defs.GATT_DISC_ALL_PRIM)
+
+    svcs_tuple = gatt_dec_disc_rsp(tuple_data[0], "service")
+    logging.debug("%s %r", gattc_disc_all_prim_rsp.__name__, svcs_tuple)
+
+    if store_rsp:
+        global VERIFY_VALUES
+
+        VERIFY_VALUES = []
+
+        for svc in svcs_tuple:
+            start_handle = "%04X" % (svc[0],)
```

Comment:
I've noticed PTS does not verify start, end handles so this is probably not needed